spec/game_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rspec'
+require 'pry'
+require './lib/card'
+require './lib/deck'
+require './lib/player'
+require './lib/turn'
+require './lib/game'
+
+RSpec.describe Game do
```

Comment:
To address your question in the submission form: normally we would want to test every method in a class; this is the exception since we did not require you to test methods that require command line input/output